s4e-backend/src/main/java/pl/cyfronet/s4e/service/InvitationService.java

```diff
@@ -56,12 +56,12 @@ public String createInvitationFrom(String email, String institutionSlug, boolean
     }
 
     @Transactional(rollbackFor = NotFoundException.class)
-    public void confirmInvitation(String token) throws NotFoundException {
+    public void confirmInvitation(String userEmail, String token) throws NotFoundException {
```

Comment:
Great change, it's better to fetch the AppUserDetails in controller layer and inject it into the service layer.